tasks/installation.yml

```diff
@@ -12,11 +12,44 @@
 
 - name: install graphite from pip
   pip:
-    executable: "/opt/rh/rh-python36/root/usr/bin/pip3.6"
+    executable: '{{ graphite_python_pip_executable }}'
     state: present
     name: "{{ item }}"
   loop: '{{ pip_packages }}'
   register: graphite_register_pip_installed
   until: graphite_register_pip_installed is success
   retries: 10
   delay: 2
+  when: graphite_use_pip
+
+- name: install graphite from source
```

Comment:
It might make sense to move the git urls to `defaults/main.yml` and use a loop? Do these modules depend on each other or what's the reason for using `block` here?